Cargo.lock

```diff
@@ -2152,7 +2257,7 @@ version = "1.12.0"
 dependencies = [
  "jni 0.10.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "panic_hook 0.1.0",
- "parity-ethereum 2.2.1",
+ "parity-ethereum 2.3.0",
```

Comment:
what happened here? Did you merge `Cargo.lock` from master?

Reply:
Also building this branch leaves your `Cargo.lock` with unstaged changes.